bundler/lib/bundler/rubygems_integration.rb

```diff
@@ -447,6 +449,12 @@ def find_name(name)
     def default_stubs
       Gem::Specification.default_stubs("*.gemspec")
     end
+
+    private
+
+    def gem_class
+      class << Gem; self; end
+    end
```

Comment:
Just curious: is this part of the fix too, or just `redefine_method` of `gem_class`?

Reply:
Nah, I just noticed that we needed to find this class in several different places so I moved the logic to a separate method in 758f6615d6da5ac6d18fd4716b1262fcdd3c2035. The real fix is redefining `Gem.load_plugins` to only load plugins of gems inside the bundle.

Reply:
I see. Thanks!